File: BILLSToPAY.ApplicationService/Model/AccountModel.cs
using System;
namespace BILLSToPAY.ApplicationService.Model
{
    public class AccountModel
    {
        public Guid? Id { get; set; }
        public string Name { get;  set; }
        public decimal? OriginalValue { get;  set; }
        public DateTime? PaymentDate { get;  set; }
        public DateTime? DueDate { get;  set; }
        public decimal? CorrectedValue { get;  set; }
        public int? NumberOfDaysLate { get;  set; }
        public RuleModel Rule { get; set; }
    }
}

File: src/web/src/app/model/account.model.ts
import { Interface } from "readline";
import { RuleModel } from './rule.model';

export interface accountModel {
  id: string,
  name: string,
  originalValue: number,
  paymentDate: Date,
  dueDate: Date,
  correctedValue: number,
  numberOfDaysLate: number,
  rule: RuleModel
}
